src/doc/book/lifetimes.md

```diff
@@ -56,7 +56,7 @@ To fix this, we have to make sure that step four never happens after step
 three. The ownership system in Rust does this through a concept called
 lifetimes, which describe the scope that a reference is valid for.
 
-*Note* It's important to understand that lifetimes are _descriptive_ not _prescriptive_. This means that the lifetimes of references are determined by the code, not by the lifetime annotations. The annotations, however, point out the lifetimes to the compiler in case it can't figure them out by itself.
+**Note** It's important to understand that lifetime annotations are _descriptive_ not _prescriptive_. This means that the lifetimes of references are determined by the code, not by the annotations. The annotations, however, point out the lifetimes to the compiler in case it can't figure them out by itself.
```

Comment:
That's better, but I also think the phrase "the lifetimes of references" should also avoid the overloaded term "lifetime" (just as it says "the annotations" a bit later). This paragraph is clarifying a confusion between the two meanings of the word lifetime (one I've see a lot) so it should avoid anything which could be misinterpreted through the lens of that confusion. Perhaps something along the lines of "how long references are valid is determined by [...]".